src/main/java/com/glisco/isometricrenders/widget/DynamicLabelComponent.java

```diff
@@ -32,13 +32,15 @@ public DynamicLabelComponent shadow(boolean shadow) {
     }
 
     @Override
-    protected void applyHorizontalContentSizing(Sizing sizing) {
+    protected int determineHorizontalContentSize(Sizing sizing) {
         this.width = 100;
```

Comment:
This needs to go. The method was renamed and the return type precisely because the component no longer applies the size itself - it returns it to the layout engine which may then apply or discard it